Expensify/App
# Fix navigating to workspace by preloading betas

### Details


### Fixed Issues
Issue uncovered while testing https://github.com/Expensify/App/pull/5439 in stg

### Tests
- Ignore for QA: remove `isDevelopment() ||` from here https://github.com/Expensify/App/blob/0e6fdbf0e3b9f87572701c90c78b2ab73540b3ff/src/libs/Permissions.js#L11
- Make sure you are signed out of newDot
- Create a new account on oldDot
- Go to Settings > Policies > Group
- Create a free policy
- Check newDot opens and shows the workflow page
- Create another free policy
- Check newDot opens and shows the workflow page
- Sign out of newDot
- Refresh oldDot
- Click on one of the free policies
- Check newDot opens and shows the workflow page
- Click on one of the free policies
- Check newDot opens and shows the workflow page

### Tested On


### Screenshots
![image](https://user-images.githubusercontent.com/521248/134494721-33550ebf-5bf7-415a-bfb3-1501ac59c7d0.png)
